include/circt/Dialect/LLHD/IR/ValueOps.td

```diff
@@ -34,34 +34,33 @@ def LLHD_ConstOp : LLHD_Op<"const", [ConstantLike, NoSideEffect]> {
   let hasFolder = 1;
 }
 
-def LLHD_VecOp : LLHD_Op<"vec", [
+def LLHD_ArrayOp : LLHD_Op<"array", [
     NoSideEffect,
     TypesMatchWith<
       "types and number of 'values' have to match the length and type of the "
-      "'result' vector",
+      "'result' array",
       "result", "values",
-      "std::vector<Type>($_self.cast<ShapedType>().getNumElements(), "
-      "$_self.cast<ShapedType>().getElementType())">
+      "std::vector<Type>($_self.cast<llhd::ArrayType>().getLength(), "
+      "$_self.cast<llhd::ArrayType>().getElementType())">
   ]> {
-  let summary = "Create a vector from a list of values.";
+  let summary = "Create an array from a list of values.";
   let description = [{
-    The `llhd.vec` operation allows to create a vector from a list of
-    SSA-values. This allows for more flexibility compared to only using
-    `std.constant` and the vector dialect operations.
+    The `llhd.array` operation allows to create an array from a list of
+    SSA-values.
 
     Example:
 
     ```mlir
     %c1 = llhd.const 1 : i32
     %c2 = llhd.const 2 : i32
     %c3 = llhd.const 3 : i32
-    %vec = llhd.vec %c1, %c2, %c3 : vector<3xi32>
-    %elem = vector.extractelement %vec[%c1 : i32] : vector<3xi32>
+    %array = llhd.array %c1, %c2, %c3 : !llhd.array<3xi32>
+    %elem = llhd.extract_element %array, 0 : vector<3xi32> -> i32
```

Comment:
This should probably be `!llhd.array<3xi32> -> i32`.

Reply:
Oh yes. Thanks!